src/utils/get-package-scope.js

```diff
@@ -0,0 +1,11 @@
+const { existsSync } = require('fs');
```

Comment:
Why synchronous? Can we use async here?

Reply:
Same answer as above. Note this is no different to require :)